draft-ietf-ppm-dap.md

```diff
@@ -1395,10 +1400,13 @@ across vast numbers of reports and preparation of large histograms, may be
 better suited for the asynchronous model. For use cases where datastore
 performance is a concern, the synchronous model may be better suited.
 
-In general, reports cannot be aggregated until the Collector specifies an
-aggregation parameter. However, in some situations it is possible to begin
-aggregation as soon as reports arrive. For example, Prio3 has just one valid
-aggregation parameter (the empty string).
+In general, reports cannot be aggregated until the Collector specifies a query
+and an aggregation parameter. However, depending on the VDAF and batch mode in
+use, it is often possible to begin aggregation as soon as reports arrive. For
+example, Prio3 has just one valid aggregation parameter (the empty string), and
+thus allows for eager aggregation; and both the time-interval and
```

Comment:
Aligns better with discussion on issue.

```suggestion
thus allows for streaming aggregation; and both the time-interval and
```

Reply:
Hmm, I'm not so sure. I think there are two concepts at play:

* "All-at-once aggregation" vs "streaming aggregation": this refers to whether aggregation happens "all at once", after recovering all of the relevant output shares (as was specified prior to this PR); or if aggregation happens in a "streaming" fashion as output shares are recovered (as is specified in this PR, and implemented in all DAP implementations).
* "Eager aggregation" vs "lazy aggregation": this refers to whether the aggregation process occurs "eagerly" as reports arrive, or if it is performed "lazily" once a collection request arrives. (Lazy aggregation is required if the Collector needs to choose an aggregation parameter.) Which of these is used is dependent on the implementation, both before & after this PR.

Reply:
You're right, thanks for clarifying.